app/helpers/audits_helper.rb

```diff
@@ -32,10 +33,11 @@ def audit_title(audit)
         (id_to_label audit.audited_changes.keys[0], audit.audited_changes.values[0]).to_s
       else
         name = audit.auditable_name.blank? ? audit.revision.to_label : audit.auditable_name
-        name += " / #{audit.associated_name}" if audit.associated_id && !audit.associated_name.blank?
+        name += " / #{audit.associated_name}" if audit.associated_id && !audit.associated_name.blank? unless type_name == 'Interface'
```

Comment:
Could you please combine this condition into the same if? i.e. 
```ruby
if audit.associated_id && !audit.associated_name.blank? && type_name != 'Interface'
```

Reply:
Ouch, sure.